packages/fxa-auth-server/lib/routes/subscriptions/stripe.ts

```diff
@@ -503,6 +511,25 @@ export class StripeHandler {
         metricsContext,
       } = request.payload as Record<string, string>;
 
+      // Make sure to clean up any subscriptions that may be hanging with no payment
+      const existingSubscription =
+        this.stripeHelper.findCustomerSubscriptionByPlanId(customer, priceId);
+      if (
+        existingSubscription &&
+        existingSubscription.status === 'incomplete'
```

Comment:
Why not `existingSubscription?.status === 'incomplete'`?

Reply:
Sure. Updated.